frontend/src/metabase/admin/datamodel/metadata/components/SemanticTypeAndTargetPicker/SemanticTypeAndTargetPicker.tsx

```diff
@@ -63,6 +63,7 @@ const SemanticTypeAndTargetPicker = ({
     >
       <SemanticTypePicker
         className={className}
+        field={field.getPlainObject()}
```

Comment:
Does this create a new object on every invocation? Can the `SemanticTypePicker` do this call instead to avoid re-renders?

Reply:
> Does this create a new object on every invocation? 

It does not

> Can the `SemanticTypePicker` do this call instead to avoid re-renders?

`SemanticTypePicker` needs to use `Field` from `metabase/types/api` (and not `metabase-lib/v1`) for compatibility with other places where [it will be used](https://github.com/metabase/metabase/pull/55539).